jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -184,8 +187,8 @@ public void setAllowInsecureRegistries(boolean allowInsecureRegistries) {
     this.allowInsecureRegistries.set(allowInsecureRegistries);
   }
 
-  public void setExtraDirectory(Path extraDirectory) {
-    this.extraDirectory.set(extraDirectory);
+  public void setExtraDirectory(String extraDirectory) {
+    this.extraDirectory.set(projectDir.resolve(extraDirectory));
```

Comment:
So this I'm not too sure of. The way I changed it here means that people can only configure paths relative to the project directory, which makes things easier in general, but also doesn't let people add absolute paths (maybe that's ok?)

Reply:
I think we can just have this take a `File`?

Reply:
That way users can use the `file` method in Gradle.